samples/bot-adaptive-card-actions/csharp/AdaptiveCardActions/Bots/AdaptiveActionsBot.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using AdaptiveCards.Templating;
+using System;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.AspNetCore.Mvc;
+using AdaptiveCards;
+using System.Net.Mime;
+
+namespace Microsoft.BotBuilderSamples
+{
+    // This bot will respond to the user's input with suggested actions.
+    // Suggested actions enable your bot to present buttons that the user
+    // can tap to provide input. 
+    public class AdaptiveActionsBot : ActivityHandler
+    {
+        public string commandString = "Please use one of these commands: **1** for  Adaptive Card Actions, **2** for Bot Suggested Actions and **3** for Toggle Visible Card";
+
+        protected override async Task OnMembersAddedAsync(IList<ChannelAccount> membersAdded, ITurnContext<IConversationUpdateActivity> turnContext, CancellationToken cancellationToken)
+        {
+            // Send a welcome message to the user and tell them what actions they may perform to use this bot
+            var welcomeText = "Hello and Welcome!";
+            await turnContext.SendActivityAsync(MessageFactory.Text(welcomeText), cancellationToken);
+            await turnContext.SendActivityAsync(MessageFactory.Text(commandString), cancellationToken);
+        }
+        protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivity> turnContext, CancellationToken cancellationToken)
+        {
+            if (turnContext.Activity.Text != null)
+            {
+                // Extract the text from the message activity the user sent.
+                var text = turnContext.Activity.Text.ToLowerInvariant();
+
+                if (text.Contains("1"))
+                {
+                    string[] path = { ".", "Cards", "AdaptiveCardActions.json" };
+                    var adaptiveCardForPersonalScope = GetFirstOptionsAdaptiveCard(path, turnContext.Activity.From.Name);
+                    await turnContext.SendActivityAsync(MessageFactory.Attachment(adaptiveCardForPersonalScope), cancellationToken);
+                }
+                else if (text.Contains("2"))
+                {
+                    //Respond to the user.
+                    await turnContext.SendActivityAsync("Please Enter a color from the suggested action choices", cancellationToken: cancellationToken);
+                    await SendSuggestedActionsAsync(turnContext, cancellationToken);
+
+                }
+                else if (text.Contains("3"))
+                {
+                    string[] path = { ".", "Cards", "ToggleVisibleCard.json" };
+                    var adaptiveCardForPersonalScope = GetFirstOptionsAdaptiveCard(path, turnContext.Activity.From.Name);
```

Comment:
yes, we are using, instead of number we are name accordingly.
<img width="606" alt="image" src="https://user-images.githubusercontent.com/109147706/210224712-b7767b03-d1b3-4c9a-8a6e-afca87e9f610.png">